### PR TITLE
Improve location formatting, fix cassettes

### PR DESCRIPTION
**Pull request by valgur**
Follow-up for #372.
The location info output was printing `{}` placeholders instead of values and the cassettes were not updated after the final bugfix.
I also tweaked the location info logging further by adding the bounding box extents in km and the box coordinates in a clearer format. So, for example, `sentinelsat --location Metz` prints
```
Querying location: 'Metz, Moselle, Grand Est, France métropolitaine, France' with 9.8 x 8.8 km, 49.060824, 6.136004 to 49.148795, 6.256451 bounding box
```
The box size info is useful since the Nominatim results are not always perfect, i.e. `sentinelsat --location Estonia` yields a bounding box halfway in Poland
```
Querying location: 'Eesti', 807.5 x 1556.7 km, 51.752378, 18.331908 to 65.752378, 32.331908
```

### CODE REVIEW

File: sentinelsat/scripts/cli.py [hunk 1]
```diff
@@ -212,8 +213,21 @@ def cli(
         search_kwargs.update((x.split("=") for x in query))
 
     if location is not None:
-        wkt, place_name = placename_to_wkt(location)
-        logger.info("The location we are querying is: '%s', '%s'".format(place_name, wkt))
+        wkt, info = placename_to_wkt(location)
+        minX, minY, maxX, maxY = info["bbox"]
+        r = 6371  # average radius, km
```
Comment (Fernerkundung) [inline sentinelsat/scripts/cli.py:218]: Can you extend the comment how this value is derived and what makes it "average"?

File: sentinelsat/sentinel.py [hunk 2]
```diff
@@ -1149,15 +1149,15 @@ def geojson_to_wkt(geojson_obj, feature_number=0, decimals=4):
...
     decimals : int, optional
         Number of decimal figures after point to round coordinate to. Defaults to 4 (about 10
         meters).
 
     Returns
     -------
-    polygon coordinates
-        string of comma separated coordinate tuples (lon, lat) to be used by SentinelAPI
+    str
+        Well-Known Text string corresponding to the GeoJSON geometry.
```
Comment (Fernerkundung) [inline sentinelsat/sentinel.py:1160]: Niggle: A WKT is a geometry representation that in itself is not related to GeoJSON, except that they both refer to spatial geometries. The phrasing sounds like this is somehow a WKT+GeoJSON string.

File: sentinelsat/sentinel.py [hunk 3]
```diff
@@ -1427,33 +1427,42 @@ def _parse_odata_response(product):
...
+    ------
+    ValueError
+        If no matches to the place name were found.
+
     Returns
     -------
-    full name and coordinates of the queried placename
-        list in the form [wkt string in form 'ENVELOPE(minX, maxX, maxY, minY)', string placeinfo]
+    wkt_envelope : str
+        Bounding box of the location as an 'ENVELOPE(minX, maxX, maxY, minY)' WKT string.
```
Comment (Fernerkundung) [inline sentinelsat/sentinel.py:1447]: A bounding box and an evenlope *can* be identical but don't have to be. Can we use a single terminology throughout the codebase?

The Envelope vs. Extent definitions can be rather confusing, see for instance PostGIS [ST_Envelope](https://postgis.net/docs/ST_Envelope.html) vs. [ST_Extent](https://postgis.net/docs/ST_Extent.html) (which I find to be a bad pattern).

File: sentinelsat/sentinel.py [hunk 4]
```diff
@@ -1427,33 +1427,42 @@ def _parse_odata_response(product):
...
     )
-    # Check that the response from Openstreetmapserver has status code 2xx
-    # and that the response is valid JSON.
     rqst.raise_for_status()
-    jsonlist = rqst.json()
-    if len(jsonlist["features"]) == 0:
-        raise ValueError('Unable to find a matching location for "{}"'.format(placename))
+    features = rqst.json()["features"]
+    if len(features) == 0:
+        raise ValueError('Unable to find a matching location for "{}"'.format(place_name))
```
Comment (Fernerkundung) [inline sentinelsat/sentinel.py:1459]: This would also be triggered if the nominatim endpoint is unavailable, wouldn't it? In that case the error message might be slightly confusing.

File: sentinelsat/sentinel.py [hunk 5]
```diff
@@ -1427,33 +1427,42 @@ def _parse_odata_response(product):
...
     # Get the First result's bounding box and description.
-    feature = jsonlist["features"][0]
+    feature = features[0]
     minX, minY, maxX, maxY = feature["bbox"]
-    footprint = "ENVELOPE({}, {}, {}, {})".format(minX, maxX, maxY, minY)
-    placeinfo = feature["properties"]["display_name"]
-    return footprint, placeinfo
+    # ENVELOPE is a non-standard WKT format supported by Solr
+    # https://lucene.apache.org/solr/guide/6_6/spatial-search.html#SpatialSearch-BBoxField
```
Comment (Fernerkundung) [inline sentinelsat/sentinel.py:1464]: ❤️

File: sentinelsat/scripts/cli.py (hunk 1 above)
Comment (valgur) [reply sentinelsat/scripts/cli.py:218]: At this level of precision (which is more than enough for this simple use case) pretty much all ways of computing the average radius of the Earth round to that value. https://en.wikipedia.org/wiki/Earth_radius

File: sentinelsat/sentinel.py (hunk 2 above)
Comment (valgur) [reply sentinelsat/sentinel.py:1160]: Good point. Is "Well-Known Text string representation of the geometry" better?

File: sentinelsat/sentinel.py (hunk 3 above)
Comment (valgur) [reply sentinelsat/sentinel.py:1447]: I guess there can be, but I don't see any way they could have a different meanings in our use cases?
Either way, I personally prefer "bounding box" as it sounds a bit clearer to me.

File: sentinelsat/sentinel.py (hunk 4 above)
Comment (valgur) [reply sentinelsat/sentinel.py:1459]: It should generate a HTTP error from `rqst.raise_for_status()` if Nominatim is not available.

File: sentinelsat/sentinel.py (hunk 2 above)
Comment (Fernerkundung) [reply sentinelsat/sentinel.py:1160]: Sounds good :+1:

File: sentinelsat/sentinel.py (hunk 3 above)
Comment (Fernerkundung) [reply sentinelsat/sentinel.py:1447]: Let's go with that then :+1:

**Comment by Fernerkundung** [comment]
LGTM!

If you add the changes to the changelog in this or another of your PRs I think we can get this merged right away @valgur :+1:
